conda/recipes/custreamz/meta.yaml

```
@@ -0,0 +1,36 @@
+# Copyright (c) 2018-2019, NVIDIA CORPORATION.
+
+{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') %}
+{% set minor_version =  version.split('.')[0] + '.' + version.split('.')[1] %}
+{% set git_revision_count=environ.get('GIT_DESCRIBE_NUMBER', 0) %}
+{% set py_version=environ.get('CONDA_PY', 36) %}
+
+package:
+  name: custreamz
+  version: {{ version }}
+
+build:
+  number: {{ git_revision_count }}
+  string: py{{ py_version }}_{{ git_revision_count }}
+
+requirements:
+  host:
+    - python
+  run:
+    - python
+    - streamz 
+    - cudf {{ version }}
+    - dask >=2.1.0
+    - distributed >=2.1.0
+    - python-confluent-kafka
+test:
```

Line break between requirements and test
```suggestion

test:
```